setup.py

```diff
@@ -6,7 +6,7 @@
 
 
 REQUIREMENTS = [
-    'Django>=2.2',
+    'Django>=2.2,<4',
```

Comment:
@marksweb Does this actually work? 

In the past I've found pip to be very picky with dependancy numbers. For example it doesn't know that 1.0 is 1 when I have tried in the past.

Reply:
@Aiky30 To confirm this...

```
(.venv) mwalker proj % pip install "django>=3.2,<4"
Requirement already satisfied: django<4,>=3.2 in ./.venv/lib/python3.9/site-packages (3.2.11)
Requirement already satisfied: asgiref<4,>=3.3.2 in ./.venv/lib/python3.9/site-packages (from django<4,>=3.2) (3.4.1)
Requirement already satisfied: sqlparse>=0.2.2 in ./.venv/lib/python3.9/site-packages (from django<4,>=3.2) (0.4.2)
Requirement already satisfied: pytz in ./.venv/lib/python3.9/site-packages (from django<4,>=3.2) (2021.3)
(.venv) mwalker proj % 
```